reviews/utils/review_utils.py

```diff
@@ -304,6 +304,21 @@ def notify_secretary(decision):
     msg_plain = render_to_string('mail/decision_notify.txt', params)
     send_mail(subject, msg_plain, settings.EMAIL_FROM, [secretary.email])
 
+def notify_secretary_all_decisions(review):
+    """
+    Notifies a secretary all Decisions have been made for a certain review
+    """
+    secretary = get_secretary()
+    subject = _("FETC-GW {}: alle beoordelingen toegevoegd").format(
+        review.proposal.committee_prefixed_refnum(),
+    )
```

Comment:
You're translating the string, but not adding any translations...

Honestly, as the email is not translated, I'd just drop the `_([..])` tag altogether